recipes-devtools/bats-suite/files/openxt-bats-suite.initscript

```diff
@@ -0,0 +1,22 @@
+#! /bin/sh
+set -e
+
+# Source sysvinit rcS.d defaults.
+. /etc/default/rcS
+
+# Source sysvinit functions library.
+. /etc/init.d/functions
+
+# /etc/init.d/openxt-bats-suite: run the OpenXT bats test-suite.
+
+LOG_FILE="/storage/openxt-bats-suite.log"
```

Comment:
Why not in `/var/log/`?

Reply:
IMO it's good to avoid `/var/log`, to be able to test log rotation and stuff.

Reply:
The log file is additional, everything is already logged to `/var/log` through `syslog`. @jean-edouard has a point though, this hits a component we may want to test.

Reply:
I am not sure how putting the file into /var/log conflicts with log rotation. Unless it has changed we have a hook in rsyslog that rotates only the files it is writing to based on size, so the only issue I could see is that the log volume filled up though that might be another good test in and of itself.